src/NzbDrone.Core/Datastore/DbFactory.cs

```diff
@@ -72,7 +84,30 @@ public IDatabase Create(MigrationContext migrationContext)
                     }
             }
 
-            _migrationController.Migrate(connectionString, migrationContext);
+            try
+            {
+                _migrationController.Migrate(connectionString, migrationContext);
+            }
+            catch (SQLiteException ex)
+            {
+                var fileName = _connectionStringFactory.GetDatabasePath(connectionString);
+
+                if (migrationContext.MigrationType == MigrationType.Log)
+                {
+                    Logger.Warn(ex, "Logging database is corrupt, attempting to recreate it automatically");
+
+                    _diskProvider.DeleteFile(fileName);
+                    _diskProvider.DeleteFile(fileName + "-shm");
```

Comment:
euhm... the corruption happens on mac,... which doesn't use wal, instead -journal.